wasm/wasm.go

```diff
@@ -107,6 +108,11 @@ type Function interface {
 	Call(ctx context.Context, params ...uint64) ([]uint64, error)
 }
 
+type Global interface {
+	Value() uint64
```

Comment:
What I was thinking was `Get() uint64` and `Set(uint64) bool` where the latter can return bool false which means it is unsupported for any reason including configuration or the global isn't mutable.

```suggestion
	Get() uint64
```